pkg/admission/machines.go

```diff
@@ -5,16 +5,13 @@ import (
 	"fmt"
 
 	"github.com/golang/glog"
+	admissionv1beta1 "k8s.io/api/admission/v1beta1"
```

Comment:
k8s imports are usually in the same import group

Reply:
See comment below, but will do so in future. Would like a code style document for it.